data/json/monsters/zed_skeletal.json

```diff
@@ -52,6 +52,37 @@
     "armor": { "bash": 12, "cut": 30, "acid": 1, "bullet": 30, "electric": 4 },
     "extend": { "weakpoint_sets": [ "wps_bone_armor" ], "families": [ "prof_wp_skeleton" ] }
   },
+  {
+    "id": "mon_skeleton_electric",
+    "type": "MONSTER",
+    "name": { "str": "skeletal shocker" },
+    "description": "Heavy, jagged bone plates have grown out of the surface of this zombie's skin.  Underneath, its flesh glows and crackles with the occasional jolt of electricity.",
+    "copy-from": "mon_zombie_base",
+    "diff": 15,
+    "hp": 105,
+    "speed": 60,
+    "material": [ "bone" ],
+    "color": "blue_white",
+    "melee_damage": [ { "damage_type": "electric", "amount": 12 }, { "damage_type": "cut", "amount": 3 } ],
+    "luminance": 8,
+    "bleed_rate": 0,
+    "harvest": "mr_bones",
+    "special_attacks": [
+      { "id": "grab", "cooldown": 21 },
+      { "id": "scratch_humanoid" },
+      { "id": "bite_humanoid", "cooldown": 5 },
+      [ "PARROT", 8 ],
+      [ "SHOCKSTORM", 25 ]
```

Comment:
So now all electrick zombies lost schockstorm abbility, except for skeletal shocker. He still is undead Zeus.

Reply:
Should be fixed by #74693.